components-core/src/main/java/com/adyen/checkout/components/core/internal/util/DateUtils.kt

```diff
@@ -78,4 +79,20 @@ object DateUtils {
             null
         }
     }
+
+    /**
+     * Format server date pattern to regular date pattern (30/03/2023).
+     *
+     * @param date Date to be formatted
+     * @param shopperLocale
+     * @param dateFormat Date pattern
+     */
+    fun formatDateToString(
+        date: Date,
```

Comment:
Is `Date` working well with different time zones?

Reply:
Creating a `Date()` instance, returns the current date and time set on the device. Are there other points I should pay attention to?